#!/usr/bin/env python
PACKAGE = "uwb_init_trigger"
CONFIG_FILE_PREFIX = "dyn_config"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()
gen.add("uwb_init_reset",         bool_t,   0, "A Boolean parameter",  False)
gen.add("uwb_init_start",         bool_t,   0, "A Boolean parameter",  False)
gen.add("uwb_init_initialize",    bool_t,   0, "A Boolean parameter",  False)
gen.add("uwb_init_compute_wps",   bool_t,   0, "A Boolean parameter",  False)
gen.add("uwb_init_refine",        bool_t,   0, "A Boolean parameter",  False)


exit(gen.generate(PACKAGE, PACKAGE, CONFIG_FILE_PREFIX))
